collection-scripts/gather

```diff
@@ -71,4 +71,7 @@ done
 # Collect v2v details
 /usr/bin/gather_v2v
 
+# Collect image details
```

Comment:
Please refer to the issue here so we know what to remove once it is fixed.

Reply:
Done